dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -310,18 +327,16 @@ public Response toolsGet(String id, String alias, String toolClass, String descr
         int offsetInteger = 0;
```

Comment:
I logged a separate ticket for this, #4739, but I think you could move line 327 to before 318 pretty safely. Or handle it in a separate PR.

Also the Swagger UI still showing 1,000 as the default limit -- is there a ticket to fix that?

Reply:
I'll add it to https://ucsc-cgl.atlassian.net/browse/SEAB-3936